afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/EventsBus.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage;
+
+import com.powsybl.afs.storage.events.AppStorageListener;
+import com.powsybl.afs.storage.events.NodeEvent;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+public interface EventsBus {
```

Comment:
What is the point of topics if the listener listen all events ?
There should be listener per topic (something like `void addListener(String topic, AppStorageListener l)`.

Reply:
Alright, this will be done in the next PR.